### DISCUSSION TITLE
led matrix range of colors

### DISCUSSION THREAD

**Discussion by MichaelVerschaeve** [pybricks/support#963] [Q&A]
Haven't tested yet, but from code, it appears that any specified color is mapped by its hue to one of 11 colors (none/black, white, the 3 primary colors, 3 secondary colors and half of the tertiary colors (orange, spring green, violet)) and then brightness is determined...
My question, why is the other half of the tertiary colors (chartreuse green, azure, rose) not supported ?
I'm assuming it is because of the firmware on the led matrix, is there another reason?
Any chance this can be circumvented, worked around ?

**Comment by dlech** [comment]
It is because of the firmware on the LED matrix. We are not aware of a way to work around it.
